autoload/neomake.vim

```diff
@@ -645,7 +648,22 @@ function! s:AddExprCallback(jobinfo, prev_index) abort
         let index += 1
 
         let before = copy(entry)
-        call call(s:postprocess, [entry], maker)
+        if type(s:postprocess) == type([])
+            let s = s:postprocess
+            for s:f in s:postprocess
+                if type(s:f) == type({})
+                    let s:this = extend(copy(s:f), {'maker': maker})
+                    call call(s:f.fn, [entry], s:this)
+                else
+                    let s:this = {'maker': maker}
+                    call call(s:f, [entry], s:this)
+                endif
+            endfor
+            " TODO: abort on first invalid entry already?!
```

Comment:
Ah okay, I see.  Removing the entry here would require duplication, right?  I think it's fine to just let it fall through to the next set of branches since that's how it worked before and `remove_invalid_entries` might still apply without post-processors.

Since it's a list of processors now, passing through `entry != before` might help debug a processor that was affected by a prior processor.